public/main.js

```diff
@@ -0,0 +1,114 @@
+//ИМПОРТ КЛАССОВ, ПОЧЕМУ ТО, СДЕЛАТЬ НЕ ВЫШЛО. бЫЛО ПЕРЕПРОБЫВАНО ОЧЕНЬ МНОГО. ТАК ЧТО, ИЗВИНИТЕ, НО ОНИ ЗДЕСЬ
+const search_btn = document.getElementById('search_btn');
+const parentBlock = document.getElementById('music_list');
+const libraryStorage = document.getElementById('libraryStorage');
+//ЗАПРОС НА СЕРВЕР
+class HttpRequests {
+    constructor(apiKey, api) {
+        this.requestURL = '';
+        this.apiKey = apiKey;
+        this.api = api;
+    }
+    async sendRequests(url, data) {
+        this.response = await fetch(url, {
+            method: data.method,
+            body: data.body
+        });
+        return await this.response.json();
+    }
+    async searchTrack(track) {
+        this.requestURL = `${this.api}/2.0/?method=track.search&track=${track}&api_key=${this.apiKey}&limit=20&format=json`;
+        return await this.sendRequests(this.requestURL, { method: "GET" });
+    }
+}
+// ОТРИСОВКА ДАННЫХ С СЕРВЕРА
+class Track {
+    constructor() {
+        this.trek = {
+            image: '',
+            treckname: '',
+            author: '',
+        };
+    }
+    addTrack(trek) {
+        this.trek = trek;
```

Comment:
Стоит называть переменные корректно.
В крайнем случае можно воспользоваться переводчиком

```suggestion
        this.track= track;
```